docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+.. include:: ../common/common_definitions.rst
+
+.. _relying-party-solution:
+
+Relying Party Solution
+++++++++++++++++++++++
+
+This document defines the implementation of the online presentation and verification flow of a credential (PID or (Q)EAA), in accordance with the "OpenID for Verifiable Credential Presentation - draft 19" specifications [OID4VCP], in the context of a Cross-Device flow scenario.
```

Comment:
do we assume that this flow won't work in same device scenario?

Reply:
I agree with you, we can remove the Cross-Device flow scenario in this introduction.
The Authorization Request and Response Object flow will work in both same and cross device scenarios, but there are some assumptions that differs between them.
We started to create a section “Same Device Flow” section that we will complete when the situation is more clear (for example, what is the redirect uri in which the verifier has to redirect the user agent in case of multi wallet?)